### PR TITLE
PEP-8 pulp.plugins.loader.

### PR DESCRIPTION
**Pull request by rbarlow**

### CODE REVIEW

**Comment by rbarlow** [comment]
@pulpbot is lying - it seems that he just doesn't have the right version of gofer. The tests passed on my checkout.

File: server/pulp/plugins/loader/manager.py [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-# implied warranties of MERCHANTABILITY, NON-INFRINGEMENT, or FITNESS FOR A
-# PARTICULAR PURPOSE.
-# You should have received a copy of GPLv2 along with this software; if not,
-# see http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
-
-import copy
-import logging
 from gettext import gettext as _
 from pprint import pformat
+import copy
```
Comment (asmacdo) [inline server/pulp/plugins/loader/manager.py]: I would prefer this to be alphabetical.

https://github.com/pulp/pulp_docker/pull/52#discussion_r22557739

**Comment by asmacdo** [comment]
LGTM

File: server/pulp/plugins/loader/manager.py (hunk 1 above)
Comment (rbarlow) [inline server/pulp/plugins/loader/manager.py]: I also prefer them to be alphabetical, but in addition I prefer to group by the two styles of import (all from's together, all import's together, and from before import).

**Pull request merged**